docs-public/snippets.js

```diff
@@ -713,7 +721,13 @@ function openInPlunkr(listing) {
 
         appModuleImports:   $.map(directives, toModuleImport).join("\n"),
         appModules:         $.map(directives, function(dir) { return dir.import }).join(", "),
-        systemjsPackages:   $.map(directives.filter(function(dir) { return dir.module.indexOf('@angular') != 0 }), toSystemJsPackage).join("\n")
+        systemjsPackages:   $.map(directives.filter(function(dir) { return dir.module.indexOf('@angular') != 0 })
+                                            .concat(systemJsDefaultKendoPackages)
+                                            .filter(function(value, index, self) {
+                                                return self.findIndex(function(current, currentI) {
+                                                    return current.module === value.module;
+                                                }) === index;
+                                            }), toSystemJsPackage).join("\n" + Array(7).join(" "))
```

Comment:
Seems quite involved. I'd suggest moving this as a standalone statement, or at least extracting the second `filter` callback as `unique` to make it more understandable.